packages/terminal/src/browser/terminal-linkmatcher-files.ts

```diff
@@ -134,22 +134,24 @@ export class TerminalLinkmatcherFiles extends AbstractCmdClickTerminalContributi
 const pathPrefix = '(\\.\\.?|\\~)';
 const pathSeparatorClause = '\\/';
 // '":; are allowed in paths but they are often separators so ignore them
-// Also disallow \\ to prevent a catastrophic backtracking case #24798
-const excludedPathCharactersClause = '[^\\0\\s!$`&*()\\[\\]+\'":;\\\\]';
+// Also disallow \\ to prevent a catastropic backtracking case #24795
```

Comment:
@perrinjerome it looks like we actually added a typo here, it should be `catastrophic`.

Reply:
As usual with code taken from vscode, the typo is in the [source](https://github.com/microsoft/vscode/blob/b118105bf28d773fbbce683f7230d058be2f89a7/src/vs/workbench/contrib/terminal/browser/links/terminalLocalLinkDetector.ts#L37) as well 😆

Reply:
Anyway, I would merge this PR regardless and tackle this typo when I'm addressing https://github.com/eclipse-theia/theia/issues/11491, at least assuming the original contributor doesn't respond to this in a timely manner. I've noticed a few issues with the existing link behavior, and using the proposed API we should be able to take care of those.

Reply:
That's fine, I perform a typo pass before releases anyways :) I did notice incorrect behavior in https://github.com/eclipse-theia/theia/pull/11398#issuecomment-1201226161.